### ISSUE TITLE
after using replace back button results in error

### ISSUE THREAD

**Issue by elbotho** [edtr-io/edtr-io#386]
In our custom table plugin we use replace on a `ChildStateType` to switch a cell from a text plugin to a image plugin.
Right after the switch using the history undo action results in "Error: There exists no document with the given id" and all data is lost.

**Comment by inyono** [comment]
Good catch, there's something wrong with `child.replace` and the history stack. It also doesn't work correctly in `multimedia-explanation`, although it doesn't crash the app. I'll look into it.

**Comment by elbotho** [comment]
Thank you!
The credit for the catch goes to our new test group.
